op-chain-ops/deployer/deployer.go

```diff
@@ -42,19 +44,38 @@ type Deployer func(*backends.SimulatedBackend, *bind.TransactOpts, Constructor)
 
 // NewL1Backend returns a SimulatedBackend suitable for L1. It has
 // the latest L1 hardforks enabled.
-func NewL1Backend() *backends.SimulatedBackend {
-	return NewBackendWithGenesisTimestamp(0, true)
+func NewL1Backend() (*backends.SimulatedBackend, error) {
+	backend, err := NewBackendWithGenesisTimestamp(ChainID, 0, true, make(map[string]*common.Address))
+	if err != nil {
+		return nil, err
+	}
+
+	return backend, nil
 }
 
 // NewL2Backend returns a SimulatedBackend suitable for L2.
 // It has the latest L2 hardforks enabled.
-func NewL2Backend() *backends.SimulatedBackend {
-	return NewBackendWithGenesisTimestamp(0, false)
+func NewL2Backend() (*backends.SimulatedBackend, error) {
+	backend, err := NewBackendWithGenesisTimestamp(ChainID, 0, true, make(map[string]*common.Address))
```

Comment:
I noticed that `true` is passed now instead of `false`, why did you change that? iirc `true` will activate hardforks that are not yet available on L2

Reply:
Copy/paste error, thank you!